doc/wg/core/notes/core-notes-2021-02-12.md

```diff
@@ -0,0 +1,192 @@
+   - an implementaiton for an external or other flash might help surface issues or establish validity
+ - amit: it seems that there remain non-obvious tradeoffs that will need an implementation to suss out best option
+ - alistair: I'm away for the next two weeks, so if folks comment on the PR in that time then when I get back I can lean into an implementation to test out the ideas
+ - [consensus yes]
+
+## libtock-rs + Tock 2.0
+ - amit: Q's from johnathan
+   - We normally have a 7 day rule for merging significant PRs
+ - johnathan: the part I want to jump is that if everyone approves, we can merge without waiting out the window for all reviewers -- looking for approval from core to skip this for 2.0
+ - alistair: I'm not sure we should waive the policy just because we're in a rush to get things in
+ - leon: appreciate tagging me, but don't wait for me
+ - alistair: the problem is that sometimes the tagged people review quickly, so things can merge in hours and others don't get a chance to look, but maybe 7 days is too long
+ - amit: no horse in this race..
+ - phil: is there a number between 1 and 7 days that's better?
+ - alistair: 3? we should consider just changing the policy
+ - loen: 3 can be pretty short though..
+ - johnathan: okay; this is a more complex conversation than I'd hoped; let's shelve this one so we can get to the next one
+
+## libtock-rs: Better safe or better fast?
+ - johnathan: Especially down at the syscall boundary, there's a tradeoff between safety and performance
+   - e.g. can shave off two instructions in yeild-no-wait with an unsafe
+   - e.g. can be faster to trust error codes and cast rather than bounds-check
+ - johnathan: currently, anywhere where using unsafe would save code size, I've been doing it, especially if it aligns with the TRD wording, but not all reviewers seem in agreement
+ - amit: two cases, one where the core kernel is making promises about the type (e.g. that it'll be a valid error code) and others where individual capsules will be making that promise
+ - johnathan: I'm only considering the core kernel, otherwise it would be a threat model issue to trust capsules
+ - amit: exactly, so in my mind, it seems fair to trust the core kernel
+ - amit: we already trust the core kernel, e.g. it could do anything to the memory space of a process..
+ - johnathan: we could potentially categorize by how common the call is; e.g. memop is not called often
+ - alistair: we should be able to make decisions for this based on how much overhead there really is, right?
+ - johnathan: per syscall or total? e.g. adding to command will add thousands to the applicaiton
+ - alistair: right, thinking total; agree with the memop example
+ - hudson: I suspect it might be challenging to have a general policy on this; can we trust the kernel to return valid error codes, yes, but are ther _other_ issues, e.g. alignment, that I as a reviewer will need to think about?
```

Comment:
```suggestion
 - hudson: I suspect it might be challenging to have a general policy on this; can we trust the kernel to return valid error codes, yes, but are there _other_ issues, e.g. alignment, that I as a reviewer will need to think about?
```